### PR TITLE
AdYouLike Bid Adapter: add video capabilities

### PR DESCRIPTION
**Pull request by guiann**
## Description of change
Add video capability on AdYouLike bidder

- contact email of the adapter’s maintainer
- [* ] official adapter submission
guillaume.andouard@adyoulike.com

This change affect user-facing APIs:
https://github.com/prebid/prebid.github.io/pull/2759

## Other information
I've returned a Vast XML in bid response. This is fine with instream video ad but I couldn't get it to work with outstream cases. 
Is VAST XML working with outstream? I constantly get the same error with outstream  "Video bid does not have required vastUrl or renderer property".  Whereas I provide a valid vast XML.

### CODE REVIEW

**Comment by musikele** [comment]
@guiann Hi there, 
can you check the parameters specified in Readme? they don't look good. Also, is there a way to get a test ad ? 
Outstream ads do work; check 33across or appnexus bidder for a live example. 
In general the PR looks good and I can merge it.

**Comment by guiann** [comment]
Thanks musikele, I've updated the documentation. 
I guess I cannot see the outstream ad because I didn't plug a proper renderer for my sample case. Is there any demo page with a working renderer I could use for tests?

**Comment by musikele** [comment]
@guiann 
You may try the demo page that is prebid docs: https://docs.prebid.org/examples/video/outstream/pb-ve-outstream-dfp.html 

Shall I wait for you to test outstream?

**Comment by guiann** [comment]
Yes please, I'll test it further before you merge.

**Comment by musikele** [comment]
OK. i'll wait for your approval then. BTW, you can also submit outstream support in another MR if the changes are too many.

**Comment by guiann** [comment]
You're right,    Can you please merge these changes.  
I'll still test outstream cases and create a new pull request witth outstream changes if necessary. 

Thank you;

**Comment by guiann** [comment]
Tested with an outstream case with a renderer set. Everything is fine.

File: modules/adyoulikeBidAdapter.md
Comment (musikele) [inline modules/adyoulikeBidAdapter.md:55]: duplicate of the previous section

File: modules/adyoulikeBidAdapter.md
Comment (musikele) [inline modules/adyoulikeBidAdapter.md:63]: wrap this code in quotes (`"`). Also, is this a test code? I can't see any test ad with this param.

**Review by musikele — CHANGES_REQUESTED** [review]
just left two quick notes on the doc file. Please if there is a way for me to get an ad from adyoulike let me know how so i can test the whole thing.

File: modules/adyoulikeBidAdapter.md
Comment (guiann) [reply modules/adyoulikeBidAdapter.md]: I've update the placement id so you can use it with our test tool.
please go to https://fo-debug.omnitagjs.com/debug/force
Fill the campaign id  with  c7f8f3d9d66f483048b2bc40df3b231c  and submit form.
This will set a cookie to force a test video campaign response.

**Pull request merged**
